.gitignore

```
@@ -17,5 +17,4 @@ org.eclipse.jdt.groovy.core.prefs
 .mvn/wrapper/*.jar
 
 # Generated readthedocs pages
-build-docs/
-
+/docs/_build
```

It may be better to keep both, when switching between old and new branches, you can still have the old generated doc directory.

```suggestion
/docs/_build
build-docs/
```